src/registrar/admin.py

```diff
@@ -2863,10 +2866,7 @@ def change_view(self, request, object_id, form_url="", extra_context=None):
 
     def save_model(self, request, obj, form, change):
 
-        if obj.creator is not None:
-            # ---- update creator ----
-            # Set the creator field to the current admin user
-            obj.creator = request.user if request.user.is_authenticated else None
+        obj.creator = request.user if request.user.is_authenticated else None
```

Comment:
We don't want to change this here. This will change the creator on every save then. You still need to check if obj.creator is already set and not change it if it is.

Reply:
Just updated!